dftool

```diff
@@ -30,6 +30,14 @@ from util.oceanutil import (
     FeeDistributor,
 )
 
+from util.oceantestutil import (
```

Comment:
:(

This PR is not doing "what's on the box". It looks like it's doing a bunch of different things.

Please do not do this.

One github issue should outline one problem.

One PR should solve that one issue.

A PR should not be a grab bag of a bunch of things.

I see here that there are 3-5 things going on. Please create 3-5 github issues. Create a PR for each of them. And let's go through them.

We can go through them quickly because the scope for each is tight. Therefore it will be easy to review and approve.

This PR, I definitely can't approve.